collections/_version2-english/9-fork/1-basic-knowledge.md

```diff
@@ -0,0 +1,119 @@
+
+For example, the forking market is in the following state:
+ - The market has 3 possible outcomes: `Yes`, `No`, and `Invalid`
+ - The [creation bond]({{glossary_creation_bond}}) is `0.35` REP
+ - All theoretical REP is `11,000,000`
+ - 2.5% of all theoretical REP is `275,000`
+
+And assuming that the outcomes `Yes` and `No` are disputed alternately, the changes in the filled dispute bonds which are calculated from the [formula]({{glossary_dispute_bond}}) are shown below:
+<table>
+  <thead>
+    <tr>
+      <th rowspan="2" colspan="1" class="center">Dispute Round</th>
+      <th rowspan="1" colspan="3" class="center">successfully-filled Dispute Bond</th>
+    </tr>
+    <tr>
+      <th rowspan="1" colspan="1" class="center">Yes</th>
+      <th rowspan="1" colspan="1" class="center">No</th>
+      <th rowspan="1" colspan="1" class="center">Invalid</th>
+    </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td class="center" >Initial Reporting</td>
+    <td class="right" >0.35</td>
+    <td class="right" >0</td>
+    <td class="right" >0</td>
+  </tr>
+  <tr>
+    <td class="center" >Dispute Round 1</td>
+    <td class="right" >0</td>
+    <td class="right" >0.7</td>
+    <td class="right" >0</td>
+  </tr>
+  <tr>
+    <td class="center" >Dispute Round 2</td>
+    <td class="right" >1.05</td>
+    <td class="right" >0</td>
+    <td class="right" >0</td>
+  </tr>
+  <tr>
+    <td class="center" >Dispute Round 3</td>
+    <td class="right" >0</td>
+    <td class="right" >2.1</td>
+    <td class="right" >0</td>
+  </tr>
+  <tr>
+    <td  rowspan="1" colspan="4" class="center" >〜</td>
+  </tr>
+  <tr>
+    <td class="center" >Dispute Round 18</td>
+    <td class="right" >68,812.8</td>
+    <td class="right" >0</td>
+    <td class="right" >0</td>
+  </tr>
+  <tr>
+    <td class="center" >Dispute Round 19</td>
+    <td class="right" >0</td>
+    <td class="right" >137,625.6</td>
+    <td class="right" >0</td>
+  </tr>
+  <tr>
+    <td class="center" >Dispute Round 20</td>
+    <td class="right" ><b>275,251.2</b></td>
+    <td class="right" >0</td>
+    <td class="right" >0</td>
+  </tr>
+  </tbody>
+ </table>
+
+In this case, the threshold (`275,000` REP) is exceeded in Dispute Round 20. That is a fork occurs when the dispute succeeds in Round 20.
+
+## When does a fork end?
+The period from the start to the end of a fork is referred to as the [forking period]({{glossary_forking_period}}). The forking period ends when either 60 days have passed, or more than 50% of [all theoretical REP]({{glossary_all_theoretical_rep}}) is migrated to some [child universe]({{glossary_child_universe}}). Whichever child universe receives the most migrated REP by the end of the forking period becomes the [winning universe]({{glossary_winning_universe}}), the other child universes become the [losing universe]({{glossary_losing_universe}}).
```

Comment:
```suggestion
The period from the start to the end of a fork is referred to as the [forking period]({{glossary_forking_period}}). The forking period ends when either 60 days have passed, or more than 50% of [all theoretical REP]({{glossary_all_theoretical_rep}}) is migrated to one of the [child universes]({{glossary_child_universe}}). Whichever child universe receives the most migrated REP by the end of the forking period becomes the [winning universe]({{glossary_winning_universe}}), the other child universes become the [losing universe]({{glossary_losing_universe}}).
```

Reply:
I'm closing this conversation. Since those were fixed by the pull request https://github.com/msagansk/Augur.Guide/pull/28.